app/helpers/training_helper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+#= Helpers for training views
+module TrainingHelper
+  # Given a module slug, it returns the first library that has a
+  # category including that module slug. It returns nil if no such
+  # library is found.
+  def find_library_from_module_slug(module_slug)
```

Comment:
I think both of these helper methods would make more sense as instance methods of `TrainingSlide` and `TrainingModule` respectively.